sippy-ng/src/component_readiness/ReleaseSelector.js

```diff
@@ -185,6 +219,23 @@ function ReleaseSelector(props) {
               </FormControl>
             )}
           </div>
+          <div>
+            {payloadSupport && (
+              <FormControl error={payloadTagError}>
+                <InputLabel htmlFor="payloadTag">
+                  Payload Tag (optional)
+                </InputLabel>
+                <Input
+                  id="payloadTag"
+                  value={payloadTag}
+                  onChange={handlePayloadTagChange}
+                />
+                {payloadTagError && (
+                  <FormHelperText>Invalid Payload Tag</FormHelperText>
+                )}
+              </FormControl>
+            )}
+          </div>
```

Comment:
I'd be tempted to think we should validate you don't mix pull request with payload tag, but these are both very seldom used features and the end result would just be no data. Probably not worth it?